nitransforms/io/lta.py

```diff
@@ -251,15 +278,40 @@ def from_string(klass, string):
         sa["dst"] = np.asanyarray(VolumeGeometry.from_string("\n".join(lines)))
         return lt
 
+    @classmethod
+    def from_ras(cls, ras, moving=None, reference=None):
+        """Create an affine from a nitransform's RAS+ matrix."""
+        lt = cls()
+        sa = lt.structarr
+        sa["sigma"] = 1.0
+        sa["mean"] = np.zeros((3, 1), dtype="float")
+        sa["type"] = 1  # RAS2RAS
+        # Just for reference, nitransforms does not write VOX2VOX
+        # PLEASE NOTE THAT LTA USES THE "POINTS" CONVENTION, therefore
+        # the source is the reference (coordinates for which we need
+        # to find a projection) and destination is the moving image
+        # (from which data is pulled-back).
+        if reference is not None:
+            sa["src"] = np.asanyarray(VolumeGeometry.from_image(reference))
+
+        if moving is not None:
+            sa["dst"] = np.asanyarray(VolumeGeometry.from_image(moving))
+        # However, the affine needs to be inverted
+        # (i.e., it is not a pure "points" convention).
+        # This inversion is consistent with self.to_ras()
+        sa["m_L"] = np.linalg.inv(ras)
+        # to make LTA file format
+        return lt
+
 
-class LinearTransformArray(BaseLinearTransformList):
+class FSLinearTransformArray(BaseLinearTransformList):
```

Comment:
Are we conflating FreeSurfer LTAs with something like a per-volume list of affines that one might use to motion-correct a BOLD series?

Reply:
Opened #147. @effigies - would you be okay addressing this on a separate PR?